### ISSUE TITLE
COD - preklad

### ISSUE THREAD

**Issue by Caleydon** [Zasilkovna/prestashop#224]
Prosim, mozete v prekladoch module miesto "COD" pouzivat "Platba pri doručení (dobierka)"?

Ako som zistil, pouzivatelia modulu vo velkej miere nevedia co COD znamena a tak ani nevedia, ci maju tuto velbu zaskrtnut.

**Comment by MatyasKoval** [comment]
Dobrý den @Caleydon ,
děkuji za nahlášení - předpokládám, že takto se Vám zobrazuje hláška ve slovenské lokalizaci našeho modulu? 
V aktuální verzi v české lokalizace je namísto "is COD" česky "je dobírka".

 S přáním krásného dne,
Matyáš Kovaľ – Packeta E-Commerce Support

**Comment by Caleydon** [comment]
Ano, pisal som o slovenskej lokalizacii. Bolo by dobre, ak by ste si to tam doplnili.
